### DISCUSSION TITLE
Why is GitVersion including the word "tags" and doubling the version number?

### DISCUSSION THREAD

**Discussion by tfaris** [GitTools/GitVersion#4294] [Q&A] [answered]
Hi,

We are tagging our repo in the following manner:

```
1.57.0-installer-test.3
```

Locally, when the gitversion commandline tool is used, we will see a SemVer like this, which is good and expected:

```
1.57.0-installer-test.3
```

However, we use GitHub actions to automate deployment of the software on pushing that tag. When run on the GitHub actions build server, the SemVer that GitVersion generates looks like this:

```
1.57.0-tags-1-57-0-installer-test-3.1
```

With the version number (1.57.0) doubled, the literal word "tags", and a ".1" added to the end. We're not using a GitVersion.yml config, but just relying on the defaults.

Thanks!

**Comment by HHobeck** [comment, marked as answer]
Please see related issues:

- #4015
- #4186
- #4187
- #4212

I think we have a common deficit here. I invite you to help.

**Comment by tfaris** [comment]
Indeed it is the same issue.

I appreciate the quick response. It turns out that searching for "-tags" is quite difficult!
